presto-main/src/test/java/com/facebook/presto/execution/TestNodeScheduler.java

```diff
@@ -460,14 +467,25 @@ public void testTtlAwareScheduling()
         assertTrue(assignments.keySet().contains(node2));
         assertTrue(assignments.keySet().contains(node3));
 
+        // Query is estimated to take 5 hours and has been executing for 1 hour, i.e, 4 hours left
+        // No nodes will have enough TTL, so we fall back to simple node selection as per the config above
+        session = sessionWithTtlAwareSchedulingStrategyAndEstimatedExecutionTime(new Duration(5, TimeUnit.HOURS));
+        nodeSelector = nodeScheduler.createNodeSelector(session, CONNECTOR_ID);
+        queryManager.setExecutionTime(new Duration(1, TimeUnit.HOURS));
+        splits.clear();
+        for (int i = 0; i < 3; i++) {
+            splits.add(new Split(CONNECTOR_ID, TestingTransactionHandle.create(), new TestSplitRemote()));
+        }
+
+        assignments = nodeSelector.computeAssignments(splits, ImmutableList.copyOf(taskMap.values())).getAssignments();
+        assertEquals(assignments.size(), 3);
+        assertEquals(assignments.keySet().size(), 3);
+        assertTrue(assignments.keySet().contains(node1));
+        assertTrue(assignments.keySet().contains(node2));
+        assertTrue(assignments.keySet().contains(node3));
```

Comment:
nit : consider [assertj](https://github.com/assertj/assertj-examples/blob/main/assertions-examples/src/test/java/org/assertj/examples/MapAssertionsExamples.java#L84) map assertions.

Reply:
Looks like we'd need to use assertJ from Guava to be able to use assertThat with multimaps.